docs/workflows/metadata_signing.rst

```diff
@@ -0,0 +1,43 @@
+.. _metadata_signing:
+
+Metadata Signing
+================
+
+In the RPM plugin, it is possible to publish signed repositories. This enables package managers to
```

Comment:
Maybe replace it with something like this?

The RPM plugin is able to sign repository metadata using a signing service configured by an administrator. This enables package managers to verify the authenticity of metadata before installing packages referenced in that metadata. Metadata signing is enabled for all repositories that have a signing service associated with them.